mysql scripts/table_creation/appointmentDB.sql

```diff
@@ -0,0 +1,74 @@
+-- VITA Project
+-- UNL CSE Ambassadors 2017
+
+USE vita;
+
+DROP TABLE IF EXISTS AppointmentQuestionAnswer;
+DROP TABLE IF EXISTS Appointment;
+DROP TABLE IF EXISTS Location;
+DROP TABLE IF EXISTS UserAnswer;
+DROP TABLE IF EXISTS PossibleAnswer;
+DROP TABLE IF EXISTS Question;
+DROP TABLE IF EXISTS QuestionAccessories;
+
+CREATE TABLE QuestionAccessories (
```

Comment:
What do you think about QuestionDesign, QuestionStructure, QuestionDisplay, QuestionProperties, or QuestionTraits? Any of those boil your soup? I'd like to obey plurality if at all possible so maybe not the last two.